LibrarySystemBackend/LibrarySystem/LibrarySystem/Services/PublisherService.cs

```diff
@@ -26,8 +26,10 @@ public async Task<bool> AddPublisherAsync(PublisherViewModel publisherViewModel)
                     PublisherAddress = publisherViewModel.PublisherAddress,
                     PublisherName = publisherViewModel.PublisherName,
                     PublisherEmail = publisherViewModel.PublisherEmail,
-                    PublisherID = publisherViewModel.PublisherID,
-                    PublisherPhoneNumber = publisherViewModel.PublisherPhoneNumber
+                    Id = publisherViewModel.PublisherID,
```

Comment:
Do not need to add Id to Add publisher method (because it is an auto-generated value)